### DISCUSSION TITLE
clr-dg-action-overflow cell position

### DISCUSSION THREAD

**Discussion by HairFelpss** [vmware-clarity/core#76] [Q&A] [answered]
Hello, sorry if this is a basic question but I was unable to find it in documentation, it's possible to choose in which cell I want to show my action-overflow?

It's always shows at the left of the table, but I want it to be at the right of the table like this image below

![Screenshot at May 23 10-58-15](https://user-images.githubusercontent.com/28222694/169836894-efb78a43-e7a3-4d81-a19a-4a6931b803c8.png)

Thank you for your help!

**Comment by ashleyryan** [comment, marked as answer]
Since this is a question about clr-datagrid, this question is better suited for the ng-clarity repo: https://github.com/vmware-clarity/ng-clarity/discussions

But to answer your question, no, there is no way to move the action column.
